### ISSUE TITLE
Placenta: Chronic villitis

### ISSUE THREAD

**Issue by janine-pappas** [obophenotype/human-phenotype-ontology#6580]
**Preferred term label: **Chronic villitis (non-infectious)**
**Synonyms: **Villitis of unknown etiology, nonspecific chronic villitis**
**Definition (free text, please give PubMed ID) **Lymphohistiocytic inflammation of the chorionic villi of the placenta**  
PMID: **27223167**
https://pubmed.ncbi.nlm.nih.gov/27223167/
Parent term (use hpo.jax.org/app)  **HP:0011409 Abnormality of the placental membranes**
Your nano-attribution (ORCID)**  **https://orcid.org/0000-0002-9579-3626**

**Comment by pnrobinson** [comment]
@janine-pappas  thanks for this suggestion. Added the term.

**Comment by janine-pappas** [comment]
Thank you Peter! We may want to move this under the parent term of HP:0100767 Abnormal Placental Morphology
